docs/index.md

```diff
@@ -4,396 +4,182 @@
-
-Genkit currently supports server-side development in JavaScript/TypeScript
-(Node.js) with Go support in active development.
-
-Follow along with its development or make contributions of your own at its
-[GitHub repository](https://github.com/firebase/genkit/).
-
-## Key features
-
-Genkit is here to help you through every step of your AI development journey,
-from the beginning of your prototyping to monitoring in production&mdash;so
-there's a lot to talk about.
-
-To get you started, here are 10 key Genkit features we think you'll love:
-
-### 1. Many models, one interface
-
-Genkit provides plugins that give you access to popular models out of the box
-and a flexible model abstraction that makes it easy to integrate any model API
-and use community-maintained models.
-Trying out a new model is as easy as changing a single argument, but each model
-can specify custom configurations.
-
-```js
-import { gemini15Flash } from '@genkit-ai/vertexai';
-import { ollama } from 'genkitx-ollama';
-import { generate } from '@genkit-ai/ai';
-
-function flipACoin(a, b) {
-  return Math.random() > 0.5 ? a : b;
-}
-
-const result = await generate({
-  model: flipACoin(gemini15Flash, 'ollama/gemma'),
-  config: { temperature: 0.3, maxOutputTokens: 200 },
-  prompt: 'What makes you the best LLM out there?',
-});
-
-console.log(result.text());
-```
-
-### 2. Structured output
-
-Generate strongly-typed data with Genkit using Zod schema. This can help you
-analyze unstructured text, generate creative content, select tasks, and send
-results back to your app as structured type-safe objects.
-
-```js
-import { generate } from "@genkit-ai/ai";
-import { gemini15Flash } from "@genkit-ai/vertexai";
-import { z } from "zod";
-
-const CreatureSchema = z.object({
-  name: z.string().describe('the name of the creature'),
-  hitPoints: z.number().describe('hit points, between 5 and 100'),
-  attacks: z.array(z.object({
-    name: z.string(),
-    damage: z.number().describe('amount of damage, between 2 and 25'),
-  })).describe('3 attacks the creature can use')
-});
-
-const createCreature = defineFlow({
-    name: "createCreature",
-    inputSchema: z.string(),
-    outputSchema: CreatureSchema,
-  },
-  (habitat) => {
-    const result = await generate({
+Genkit is a framework designed to help you build AI-powered applications and features. It provides open source libraries for Node.js and Go, plus developer tools for testing and debugging.
+
+This documentation covers Genkit for Node.js. If you're a Go developer, see the [Genkit Go documentation](https://firebase.google.com/docs/genkit-go/get-started-go).
+
+You can deploy and run Genkit libraries anywhere Node.js is supported. It's designed to work with any generative AI model API or vector database. While we offer integrations for Firebase and Google Cloud, you can use Genkit independently of any Google services.
+
+[Get started](http:///docs/genkit/get-started){: .button}
```

Comment:
This button isn't rendering for me, in Github

Reply:
It won't, you gotta look at the DevSite staging link: https://firebase.devsite.corp.google.com/docs/genkit